tools/util.py

```diff
@@ -130,16 +130,27 @@ def touch(fname):
         open(fname, 'a').close()
 
 
+def str_to_list(value):
+    return value if type(value) is list else [value]
+
+
 # Recursive search for files of certain extensions.
 # (Recursive glob doesn't exist in python 2.7.)
-def find_exts(directory, *extensions):
+def find_exts(directories, extensions, skip=[]):
+    directories = str_to_list(directories)
+    extensions = str_to_list(extensions)
```

Comment:
Just have directories and extensions always be a list.
```
isinstance(directories, list)
isinstance(extensions, list)
```

Reply:
✌️